qqmlsortfilterproxymodel_plugin.cpp

```diff
@@ -0,0 +1,31 @@
+#include "qqmlsortfilterproxymodel_plugin.h"
+
+#include <qqml.h>
+
+#include "qqmlsortfilterproxymodel.h"
+#include "filter.h"
+#include "sorter.h"
+
+using namespace qqsfpm;
+
+void QQmlsortfilterproxymodelPlugin::registerTypes(const char *uri)
+{   
+    //Filter
+    qmlRegisterUncreatableType<Filter>("SortFilterProxyModel", 0, 2, "Filter", "Filter is an abstract class");
```

Comment:
This was actually copy pasted code from: https://github.com/Proemion/SortFilterProxyModel/blob/72678663b1988cf3a2a5885f8fabb5e521e93fab/filter.cpp#L436

https://github.com/Proemion/SortFilterProxyModel/blob/72678663b1988cf3a2a5885f8fabb5e521e93fab/qqmlsortfilterproxymodel.cpp#L369

https://github.com/Proemion/SortFilterProxyModel/blob/72678663b1988cf3a2a5885f8fabb5e521e93fab/sorter.cpp#L212

So, didn't change these things.

(The commented out  //Q_COREAPP_STARTUP_FUNCTION(registerFilterTypes) , is to prevent it from registering the library twice, so the registerXTypes function from the .cpp files are never called)